backend/src/core/models/campaign.ts

```diff
@@ -104,6 +104,12 @@ export class Campaign extends Model<Campaign> {
   })
   demoMessageLimit!: number
 
+  @Column({
+    type: DataType.DATE,
+    allowNull: true,
+  })
+  visibleAt?: Date
```

Comment:
Why are we double-saving here, this field can and should be inferred from corresponding job queue records

Reply:
fixed